src/chainparams.cpp

```diff
@@ -671,6 +689,15 @@ class CDevNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0024].nThresholdMin = 60;   // 60% of 100
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0024].nFalloffCoeff = 5;    // this corresponds to 10 periods
 
+        // Deployment of basic BLS scheme
+        consensus.vDeployments[Consensus::DEPLOYMENT_BASIC_BLS_SCHEME].bit = 8;
+        consensus.vDeployments[Consensus::DEPLOYMENT_BASIC_BLS_SCHEME].nStartTime = 1625097600; // July 1st, 2021
```

Comment:
Probably doesn't matter if the same date is re-used from a previous fork, but just in case?
```suggestion
        consensus.vDeployments[Consensus::DEPLOYMENT_BASIC_BLS_SCHEME].nStartTime = 1661990400; // Sep 1st, 2022
```

Reply:
OK I will include this change in the next version